### PR TITLE
Vc707 nopcie switchable parant freedom

### PR DESCRIPTION
**Pull request by mcd500**
I rebased on master as the discussion at https://github.com/sifive/fpga-shells/pull/64
Making it selectable to build for vc707 with having HMC-PCIE board or not.

### CODE REVIEW

File: src/main/scala/shell/xilinx/VC707Shell.scala [hunk 1]
```diff
@@ -21,10 +21,10 @@ import sifive.fpgashells.ip.xilinx.{IBUFDS, PowerOnResetFPGAOnly, sdio_spi_bridg
 //vc707_sys_clock_mmcm0, vc707_sys_clock_, vc707_sys_clock_mmcm2 , vc707reset}
 import sifive.fpgashells.clocks._
 //-------------------------------------------------------------------------
-// VC707Shell
+// VC707BaseShell
 //-------------------------------------------------------------------------
 
-trait HasDDR3 { this: VC707Shell =>
+trait HasDDR3 { this: VC707BaseShell =>
```
[the remark this answers is not in the document]
Comment (mcd500) [reply src/main/scala/shell/xilinx/VC707Shell.scala]: Thanks, will do. Probably this weekend. I am a bit busy the rest of this weekdays.

[the remark this answers is not in the document]
Comment (mcd500) [reply src/main/scala/shell/xilinx/VC707Shell.scala]: I was able to make a small time slot. I just pushed the changes as your comment. https://github.com/sifive/fpga-shells/pull/79/commits/099754127e4ec583222d3dc26f52472a7f9c3cc7
Thanks

**Comment by erikdanie** [comment]
Ok this is good and I have merged it!

**Comment by mcd500** [comment]
Thanks!

**Pull request merged**